bedrock/settings/base.py

```diff
@@ -65,11 +65,18 @@ def data_path(*args):
 IS_POCKET_MODE = site_mode == "Pocket"
 IS_MOZORG_MODE = not IS_POCKET_MODE
 
+db_connection_max_age_secs = config("DB_CONN_MAX_AGE", default="600", parser=int)
+db_conn_health_checks = config("DB_CONN_HEALTH_CHECKS", default="False", parser=bool)
+db_default_url = config(
+    "DATABASE_URL",
+    default=f"sqlite:////{data_path('bedrock.db')}",
+)
+
 DATABASES = {
-    "default": config(
-        "DATABASE_URL",
-        default=f"sqlite:////{data_path('bedrock.db')}",
-        parser=dj_database_url.parse,
+    "default": dj_database_url.parse(
+        db_default_url,
+        conn_max_age=db_connection_max_age_secs,
+        conn_health_checks=db_conn_health_checks,
```

Comment:
Do these get ignored if sqlite?

Reply:
Yeah i've tried it locally and they seem happy
I can push this to bedrock-test to be sure

Reply:
https://github.com/mozilla-sre-deploy/deploy-bedrock/actions/runs/9687725130

Reply:
cool, just didn't know if we needed a "if postgres" check.